base/codeinsights-db/codeinsights-db.Deployment.yaml

```diff
@@ -49,15 +53,30 @@ spec:
         - mountPath: /var/lib/postgresql/data/
           name: disk
         - mountPath: /conf
-          name: timescaledb-conf
+          name: codeinsights-conf
+      - name: codeinsights-exporter
+        image: index.docker.io/sourcegraph/postgres_exporter:insiders@sha256:50b1d8b7b71bb7eca9bf53098b1b159d3d400f5aa95751dd62d96a07f460996a
+        env:
+        - name: DATA_SOURCE_NAME
+          value: postgres://postgres:password@localhost:5432/?sslmode=disable
+        - name: PG_EXPORTER_EXTEND_QUERY_PATH
+          value: /config/code_insights_queries.yaml
```

Comment:
This file doesn't actually exist yet so the exporter will log an error. We can ignore that; I'm just adding this variable so it's available whenever we do update the exporter.